### PR TITLE
Extract

### LINKED ISSUE #34: Pipeline - Load
Upload data into the RDS

- [ ] Connect to SQL Server database on RDS
- [ ] Load data using SQL commands

We need to store the data somewhere

### LINKED ISSUE #35: Pipeline - etl_pipeline (combined)
Combine the ETL scipts so that it can be run easily

- [ ] Create a new file
- [ ] Import ETL files
- [ ] Make sure the ETL functions can be run in one script

We as engineers want to make sure that the code is easy to run from an 'outside' perspective. Only needing to run 1 script will make that easy

### PR DESCRIPTION
**Pull request by joe1606**
closes #35 
closes #34

### CODE REVIEW

File: pipeline/main.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import extract
+import transform
+import load
+
+NUMBER_OF_PLANTS = 51
```
Comment (BerkayDur) [inline pipeline/main.py:5]: Pls put into .env

File: pipeline/main.py
Comment (BerkayDur) [inline pipeline/main.py:1]: pylint this file pls

**Review by BerkayDur — DISMISSED** [review]
Some changes to improve lint score

**Review by BerkayDur — APPROVED** [review]
lebowski_thumbs

**Comment by joe1606** [comment]
> lebowski_thumbs

**Pull request merged**